styx-api-service/src/main/java/com/spotify/styx/api/BackfillResource.java

```diff
@@ -347,8 +350,10 @@ private static boolean isActiveState(RunStateData runStateData) {
       throw new RuntimeException(e);
     }
 
-    final List<Instant> processedInstants = rangeOfInstants(
-        backfill.start(), backfill.nextTrigger(), backfill.schedule());
+    final boolean reversed = backfill.start().isAfter(backfill.end());
```

Comment:
same as above

Reply:
It wouldn't change the logic, but would change the semantics a bit though. Instead of checking only `start` and `end`, that would check `start`, `nextTrigger`, and `end`.